### DISCUSSION TITLE
Any way of re-adding all installed components to the UI/updater?

### DISCUSSION THREAD

**Discussion by thefunkygibbon** [Guilouz/Creality-Helper-Script-Wiki#579] [General]
something weird happened to my printer and i had to remove/reinstall moonraker,  but despite it working again, it doesn't seem to have any of the installed components in the fluidd UI's updater (or other menus for that matter, ie the timelapse tab). 

Is there a way of re-adding these without having to uninstall/re-install?    it would be great if there was an option to reinstall each installed components rather than having to uninstall them first and face losing config etc.

**Comment by Guilouz** [comment]
No there is no way to do that except by hand.

**Reply by thefunkygibbon** [reply]
is it possible to add such things to the script? 
if not , is there any documentation (appreciate it wouldn't be yours) that explains how to do it by hand?

**Reply by jb510** [reply]
@thefunkygibbon have you removed and reinstalled Fluidd using the remove menu provided by the script? https://guilouz.github.io/Creality-Helper-Script-Wiki/helper-script/helper-script-installation/#remove-menu

**Reply by thefunkygibbon** [reply]
no i haven't attempted that yet as I didn't want to break anything else.  but I removed and reinstalled moonraker as it seemed to have randomly stopped working.  given that I struggled to get time lapse/obico running I didn't want to effectively start from scratch again if I could help it.   I guess it sounds like I need to tho
